packages/blocks/drawing/src/app.tsx

```diff
@@ -48,7 +48,7 @@ const BASE_HEIGHT = 500;
 // via file upload method) as opposed to the current approach of storing in JSON.
 
 export const App: BlockComponent<BlockEntityProperties> = ({
-  graph: { blockEntity },
+  graph: { blockEntity, readonly: graphReadOnly },
```

Comment:
Any particular reason we are using `graphReadOnly` and `readonly` in other blocks? It’d be nice to keep this consistent (one way or the other)

Reply:
The reason I used it here is because the drawing block already had readonly property before I added this new change. Users can decide to put the block in read-only mode even though the block is not in a read-only page. I renamed to `graphReadOnly` to differentiate it from the existing readonly prop the block has